_maps/RandomRuins/SpaceRuins/infested_frigate.dmm

```diff
@@ -0,0 +1,5149 @@
+	},
+/obj/item/ammo_casing/spent,
+/obj/effect/spawner/random/entertainment/arcade,
+/turf/open/floor/pod/dark,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"uC" = (
+/obj/effect/turf_decal{
+	dir = 2
+	},
+/obj/machinery/recharge_station{
+	icon_state = "borgdecon2"
+	},
+/turf/open/floor/mineral/plastitanium,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"uG" = (
+/obj/effect/turf_decal{
+	icon_state = "warningline_red";
+	dir = 6
+	},
+/obj/machinery/shower/directional/west,
+/obj/effect/turf_decal{
+	icon_state = "warningline_red";
+	dir = 4
+	},
+/mob/living/simple_animal/hostile/alien{
+	faction = list("syndicate","xenomorph")
+	},
+/turf/open/floor/pod/dark,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"vj" = (
+/obj/machinery/door/poddoor{
+	id = "Brutusexterior";
+	closingLayer = 2.65
+	},
+/obj/effect/spawner/structure/window/reinforced/plasma/plastitanium{
+	layer = 2.9
+	},
+/turf/open/floor/plating,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"vm" = (
+/obj/machinery/door/airlock/external/ruin,
+/obj/effect/mapping_helpers/airlock/locked,
+/turf/open/floor/mineral/plastitanium,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"vx" = (
+/obj/structure/cable,
+/obj/effect/decal/cleanable/dirt,
+/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden,
+/turf/open/floor/mineral/plastitanium/red,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"vz" = (
+/obj/effect/mapping_helpers/burnt_floor,
+/obj/machinery/atmospherics/components/unary/vent_pump/on,
+/obj/item/ammo_casing/spent,
+/obj/item/ammo_casing/spent,
+/turf/open/floor/mineral/plastitanium,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"vM" = (
+/obj/item/ammo_casing/spent,
+/obj/effect/decal/cleanable/dirt,
+/obj/effect/decal/cleanable/xenoblood{
+	icon_state = "xltrails_1";
+	dir = 8
+	},
+/turf/open/floor/mineral/plastitanium,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"wc" = (
+/obj/structure/cable,
+/obj/structure/alien/weeds,
+/obj/item/shard{
+	icon_state = "plastitaniumtiny"
+	},
+/obj/effect/mob_spawn/corpse/human/syndicatecommando/lessenedgear,
+/obj/structure/bed/nest,
+/mob/living/simple_animal/hostile/alien/drone{
+	del_on_death = 1;
+	loot = list(/obj/effect/gibspawner/xeno);
+	faction = list("syndicate","xenomorph")
+	},
+/turf/open/floor/plating,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"wi" = (
+/obj/effect/mapping_helpers/airlock/locked,
+/obj/structure/cable,
+/obj/effect/turf_decal{
+	icon_state = "warningline_white";
+	dir = 1
+	},
+/obj/effect/turf_decal{
+	icon_state = "warningline_white";
+	dir = 4
+	},
+/obj/effect/turf_decal{
+	icon_state = "warningline_white"
+	},
+/obj/effect/turf_decal{
+	icon_state = "warningline_white"
+	},
+/obj/effect/turf_decal{
+	icon_state = "warningline_white";
+	dir = 1
+	},
+/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden,
+/obj/machinery/door/airlock/highsecurity{
```

Comment:
For welded doors there is a helper: `/obj/effect/mapping_helpers/airlock/welded` which you can drop on top of the door to weld it. Please use this. There are a couple of these on the map.

There is also an update paths that will do this for you in the `tools/UpdatePaths/Scripts. Find '74206_airlock_weld_help' and drag it onto the UpdatePaths batch file to run it.

Reply:
Done!